src/nexusrpc/handler/_common.py

```diff
@@ -35,6 +39,12 @@ def __new__(cls, *args: Any, **kwargs: Any):
     """
     Optional header fields sent by the caller.
     """
+    task_cancellation: OperationTaskCancellation = field(
```

Comment:
Since this is abstract and cannot be instantiated directly, and since Python is not GA, and since we now have a fixated version on the Temporal side, I think we can just make this a required field